how-to/customize-workspace/client/src/framework/platform/interopbroker.ts

```diff
@@ -67,10 +67,7 @@ export function interopOverride(
 							height: 715,
 							width: 665,
 							fdc3InteropApi: "2.0",
-							url: window.location.href.replace(
-								"platform/provider.html",
-								"common/windows/intents/instance-picker.html"
-							),
+							url: "http://localhost:8080/common/windows/intents/instance-picker.html",
```

Comment:
We can't do this. The reason why it grabs the location instead of a hardcoded url is because the source can exist on different servers beyond localhost and our github/samples domain.

Reply:
half reverted, but is uses rootUrl from config instead of location.replace